### PR TITLE
chore: Use dimensions from dimen resources

### LINKED ISSUE #14: chore: Use dimensions from dimen resources.
**Describe the feature you'd like**
Currently, the xml files are using hardcoded dimensions. Store those dimensions in the dimens resource files and use them from there.

### PR DESCRIPTION
**Pull request by aman-g0yaa**
Fixes #14 
**Changes:**

Stored all the hardcoded dimensions to dimen resource file and used it from there.
 
Please make sure these boxes are checked before submitting your pull request - thank you!

- [x] Run unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into a single commit by squashing them.

Please add screenshots/GIFs here if there are any changes to the UI below.

### CODE REVIEW

**Comment by amangoyal2339** [comment]
git
This time I separately added the xml files but idea file still got added ,don't know why . please help.

**Comment by aman-g0yaa** [comment]
This time I added the files separately but still idea file got added , don't know why . please help .

**Comment by immadisairaj** [comment]
> This time I added the files separately but still idea file got added , don't know why . please help .

revert, add the files needed and force push into the same branch

File: app/src/main/res/layout/activity_info.xml [hunk 1]
```diff
@@ -20,72 +20,72 @@
                 android:id="@+id/ll_info"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
-                android:layout_margin="10dp"
+                android:layout_margin="@dimen/margin1"
```
Comment (immadisairaj) [inline app/src/main/res/layout/activity_info.xml]: do not use it as `margin1`. it has to be generalized
Not only this but all the other also

**Review by immadisairaj — CHANGES_REQUESTED** [review]

**Comment by immadisairaj** [comment]
@amangoyal2339 can you please check the build log and fix the build.

**Comment by aman-g0yaa** [comment]
generalized means ? can please give me an example.

**Comment by aman-g0yaa** [comment]
Ok so I will generalize the size not the name , Got it .
Thank you.

**Review by immadisairaj — APPROVED** [review]

**Pull request merged**